src/libraries/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/CryptoHelpers.cs

```diff
@@ -73,8 +73,15 @@ private static XmlDsigXsltTransform CreateXmlDsigXsltTransform()
             {
                 return (CryptoConfig.CreateFromName(name) ?? CreateFromKnownName(name)) as T;
             }
-            catch (Exception)
+            catch (Exception e)
             {
+#if NETCOREAPP
```

Comment:
You may need to remove the `if` since you are getting errors for the other TFM builds.

Reply:
Fixed